### ISSUE TITLE
example crashes on drawing with non-english fonts.

### ISSUE THREAD

**Issue by HinTak** [Robmaister/SharpFont#71]
On Linux, after applying my enhancement (
https://github.com/Robmaister/SharpFont/issues/69 ) , I played with loading some other fonts with the example, and can reliably crash the example with loading one of the non-english Google-Noto fonts:

```
System.ArgumentException: A null reference or invalid value was found [GDI+ status: InvalidParameter]
  at System.Drawing.GDIPlus.CheckStatus (Status status) <0x420257f0 + 0x0017b> in <filename unknown>:0 
  at System.Drawing.Bitmap..ctor (Int32 width, Int32 height, PixelFormat format) <0x420ee5b0 + 0x00053> in <filename unknown>:0 
  at System.Drawing.Bitmap..ctor (Int32 width, Int32 height) <0x420f5570 + 0x0001b> in <filename unknown>:0 
  at (wrapper remoting-invoke-with-check) System.Drawing.Bitmap:.ctor (int,int)
  at Examples.Program.RenderString (SharpFont.Library library, SharpFont.Face face, System.String text) <0x42251520 + 0x00823> in <filename unknown>:0 
  at Examples.ExampleForm.pictureBoxText_Paint (System.Object sender, System.Windows.Forms.PaintEventArgs e) <0x421b3190 + 0x00053> in <filename unknown>:0 
  at System.Windows.Forms.Control.OnPaint (System.Windows.Forms.PaintEventArgs e) <0x421a6580 + 0x00070> in <filename unknown>:0 
  at System.Windows.Forms.PictureBox.OnPaint (System.Windows.Forms.PaintEventArgs pe) <0x421b2a20 + 0x00067> in <filename unknown>:0 
  at System.Windows.Forms.Control.WmPaint (System.Windows.Forms.Message& m) <0x421a3e80 + 0x00157> in <filename unknown>:0 
  at System.Windows.Forms.Control.WndProc (System.Windows.Forms.Message& m) <0x420df4b0 + 0x001f7> in <filename unknown>:0 
  at System.Windows.Forms.Control+ControlWindowTarget.OnMessage (System.Windows.Forms.Message& m) <0x420df480 + 0x00024> in <filename unknown>:0 
  at System.Windows.Forms.Control+ControlNativeWindow.WndProc (System.Windows.Forms.Message& m) <0x420df440 + 0x0003a> in <filename unknown>:0 
  at System.Windows.Forms.NativeWindow.WndProc (IntPtr hWnd, Msg msg, IntPtr wParam, IntPtr lParam) <0x420ddc00 + 0x00334> in <filename unknown>:0 
```

How does it behave on windows? NotoSansKhmer* , for example from the Google-Noto font set.

**Comment by HinTak** [comment]
Okay, I can crash the example under wine with MS dotnet and GDI+ also, with a similar backtrace, using NotoSansKhmer-Bold.ttf .

**Comment by HinTak** [comment]
The backtrace under wine with MS dotnet and GDI+ is very similar:

```
System.ArgumentException: Parameter is not valid.
   at System.Drawing.Bitmap..ctor(Int32 width, Int32 height, PixelFormat format)
   at System.Drawing.Bitmap..ctor(Int32 width, Int32 height)
   at Examples.Program.RenderString(Library library, Face face, String text)
...
```

**Comment by Robmaister** [comment]
Is this now resolved by #69?

**Comment by HinTak** [comment]
Yes, it is fixed by https://github.com/Robmaister/SharpFont/pull/70/commits/33cdec24aed767c70bfec01b1f740b0016601f31 .

The caller can cope with a null image returned, and not paint anything if the font does not have English glyphs. I tried under both mono and wine.
